AdminWebsite/AdminWebsite/ClientApp/src/app/booking/assign-judge/assign-judge.component.html

```diff
@@ -16,13 +16,13 @@ <h2 class="govuk-error-summary__title" id="error-summary-title">
   </h2>
   <div class="govuk-error-summary__body">
     <ul class="govuk-list govuk-error-summary__list">
-      <li *ngIf="judgeName.invalid">
-        <a (click)="goToDiv('judgeName-error')">Please select a courtroom account</a>
+      <li *ngIf="!judge">
```

Comment:
Consider using templates here

Reply:
I was just using the structure that was there before. Templates could be a better fit if you want me to do it as part of this I could.

Reply:
Feel free to make improvements to the code as you find issues like these

Reply:
```html
<ng-container *ngIf="judge; else nonJudge">
      <li *ngIf="judgeDisplayNameInvalid">
        <a (click)="goToDiv('judgeDisplayName-error')">{{ constants.Error.JudgeDisplayNameErrorMsg }}</a>
      </li>
      <li *ngIf="isJudgeParticipantError">
        <a (click)="goToDiv('judgeDisplayName-error')">{{ constants.Error.JudgeParticipantErrorMsg }}</a>
      </li>
</ng-container>
<ng-template #nonJudge>
  <li>
    <a (click)="goToDiv('judgeEmail-error')">{{ constants.Error.JudgeEmailErrorMsg }}</a>
  </li>
</ng-template>
```